### PR TITLE
MVC Part 2

### PR DESCRIPTION
**Pull request by aligmic**

### CODE REVIEW

File: AcademicManagerV1/WebContent/html/SecretaryMain.html [hunk 1]
```diff
@@ -22,9 +22,9 @@
 					Welcome <br>Secretary
 				</p>
 				<div class="col-md-12">
-					<a href="courses" class="btn btn-block btn-primary">Courses</a> <a
-						href="./persons?type=students" class="btn btn-block btn-primary">Students</a>
-					<a href="./persons?type=teachers" class="btn btn-block btn-primary">Teachers</a>
+					<a href="courses?id=0" class="btn btn-block btn-primary">Courses</a> <a
```
Comment (gmetal) [inline AcademicManagerV1/WebContent/html/SecretaryMain.html]: No! Magic numbers are bad!

File: AcademicManagerV1/WebContent/html/SecretaryMain.html [hunk 2]
```diff
@@ -22,9 +22,9 @@
...
+						href="./persons?type=students&id=0" class="btn btn-block btn-primary">Students</a>
```
Comment (gmetal) [inline AcademicManagerV1/WebContent/html/SecretaryMain.html]: No! Magic numbers are bad!

File: AcademicManagerV1/WebContent/html/SecretaryMain.html [hunk 3]
```diff
@@ -22,9 +22,9 @@
...
+					<a href="./persons?type=teachers&id=0" class="btn btn-block btn-primary">Teachers</a>
```
Comment (gmetal) [inline AcademicManagerV1/WebContent/html/SecretaryMain.html]: No! Magic numbers are bad!

File: AcademicManagerV1/src/com/academic/servlets/CourseServlet.java [hunk 4]
```diff
@@ -20,21 +20,78 @@
...
 	private static final long serialVersionUID = 1L;
 
-	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
-		
+	protected void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws ServletException, IOException {
+
+		int id = Integer.parseInt(request.getParameter("id"));
+
+		if (id == 0) {
```
Comment (gmetal) [inline AcademicManagerV1/src/com/academic/servlets/CourseServlet.java]: No! Magic numbers are bad!  See [javax.servlet.ServletRequest](http://docs.oracle.com/javaee/7/api/javax/servlet/ServletRequest.html#getParameter-java.lang.String-) -  ```Returns the value of a request parameter as a String, or null if the parameter does not exist```

File: AcademicManagerV1/src/com/academic/servlets/PersonsServlet.java [hunk 5]
```diff
@@ -24,26 +24,96 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
 			throws ServletException, IOException {
 
 		String type = request.getParameter("type");
+		int id = Integer.parseInt(request.getParameter("id"));
+
+		if (id == 0) {
```
Comment (gmetal) [inline AcademicManagerV1/src/com/academic/servlets/PersonsServlet.java]: No! Magic numbers are bad!  See [javax.servlet.ServletRequest](http://docs.oracle.com/javaee/7/api/javax/servlet/ServletRequest.html#getParameter-java.lang.String-) -  ```Returns the value of a request parameter as a String, or null if the parameter does not exist```

File: AcademicManagerV1/src/com/academic/servlets/PersonsServlet.java [hunk 6]
```diff
@@ -24,26 +24,96 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
...
+		p.setBankAccount(bankAccount);
+		p.setPersonId(id);
+		personDao.update(p);
+
+		if (type.equals("Student")) {
+			response.sendRedirect("./persons?type=students&id=0");
 			return;
 		}
+
+		response.sendRedirect("./persons?type=teachers&id=0");
```
Comment (gmetal) [inline AcademicManagerV1/src/com/academic/servlets/PersonsServlet.java]: Bad magic number

**Comment by aligmic** [comment]
no more black magic numbers!

**Review by gmetal — APPROVED** [review]
LGTM

**Pull request merged**